src/automator/tasks/overview.spec.js

```diff
@@ -80,16 +89,47 @@ describe('overview task handler', function() {
     spyOn(fileStore, 'getContent')
       .and.returnValue(q.when(data.fileContent));
     backend.expectPOST('http://richtext/v0/richtext/').respond(201);
-    overview({
-      entity: 'file'
-    }, {
-      entities: {
-        file: data.fileEntity
+    overview(
+      {
+        entity: 'file'
       },
-      collab: data.collab
-    }).then(function(res) {
-      expect(res).toEqual(data.expectedNavItem);
+      {
+        entities: {
+          file: data.fileEntity
+        },
+        collab: data.collab
+      }).then(function(res) {
+        expect(res).toEqual(data.expectedNavItem);
+      });
+    scope.$digest();
+  }));
+
+  it('should update content appId', inject(function() {
+    spyOn(navStore, 'getRoot')
+      .and.returnValue(q.when(data.rootNavItem));
+    spyOn(fileStore, 'getContent')
```

Comment:
Is this still called? I don't understand why you need to call getContent when you just want to update the navItem.